applications/FluidDynamicsApplication/custom_processes/distance_modification_process.h

```diff
@@ -159,6 +160,8 @@ class KRATOS_API(FLUID_DYNAMICS_APPLICATION) DistanceModificationProcess : publi
     std::vector<unsigned int>              mModifiedDistancesIDs;
     std::vector<double>                 mModifiedDistancesValues;
     std::vector<Vector>        mModifiedElementalDistancesValues;
+    std::vector<std::string>                mDoubleVariablesList;
+    std::vector<std::string>             mComponentVariablesList;
```

Comment:
I would suggest storing those as `std::vector<const Variable<double>&>` and `std::vector<const ComponentType&>` (or `*` if the reference does not work) and initializing the vector at construction from the string input.